proto/viam/app/mltraining/v1/ml_training.proto

```diff
@@ -56,6 +59,18 @@ message SubmitTrainingJobResponse {
   string id = 1;
 }
 
+message SubmitCustomTrainingJobRequest {
+  string dataset_id = 1 [(tagger.v1.tags) = "bson:\"dataset_id\" json:\"dataset_id\""];
+  string registry_item_id = 2 [(tagger.v1.tags) = "bson:\"registry_item_id\" json:\"registry_item_id\""];
+  string organization_id = 3 [(tagger.v1.tags) = "bson:\"organization_id\" json:\"organization_id\""];
+  string model_name = 4 [(tagger.v1.tags) = "bson:\"model_name\" json:\"model_name\""];
+  string model_version = 5 [(tagger.v1.tags) = "bson:\"model_type\" json:\"model_type\""];
```

Comment:
Should the tag for this be `model_version`?

Reply:
🤦 good catch